crates/nu-cli/src/cli.rs

```diff
@@ -259,24 +246,6 @@ pub async fn cli(mut context: EvaluationContext) -> Result<(), Box<dyn Error>> {
             .scope
             .add_env_var("CMD_DURATION", format!("{:?}", cmd_start_time.elapsed()));
 
-        // Check the config to see if we need to update the path
```

Comment:
I personally disagree with reloading and applying the global config without explicit request from the user. But I am happy to discuss this